docs/getting-started/fundamentals-block-development/registration-of-a-block.md

```diff
@@ -0,0 +1,95 @@
+# Registration of a block
+
+A block is usually registered through a plugin in both the server and the client using its `block.json` metadata. 
+
+Although technically, blocks could be registered only on the client, **registering blocks in both the server and the client is a strong recommendation**, as some server-side features like Dynamic Rendering, Block Supports, Block Hooks, or Block style variations require the block to "exist" in the server and they won't work properly without a server registration of the block.
+
+For example, to enable the [capability of blocks to be styled via `theme.json`](https://developer.wordpress.org/themes/global-settings-and-styles/settings/blocks/), they need to be registered in the server, otherwise, any styles assigned to them in `theme.json` will be ignored. 
```

Comment:
```suggestion
For example, to allow a block [to be styled via `theme.json`](https://developer.wordpress.org/themes/global-settings-and-styles/settings/blocks/), it needs to be registered on the server, otherwise, any styles assigned to it in `theme.json` will be ignored. 
```